routers/api/v1/repo/issue_reaction.go

```diff
@@ -188,19 +190,20 @@ func changeIssueCommentReaction(ctx *context.APIContext, form api.EditReactionOp
 		if err != nil {
 			if models.IsErrForbiddenIssueReaction(err) {
 				ctx.Error(http.StatusForbidden, err.Error(), err)
+			} else if models.IsErrReactionAlreadyExist(err) {
+				ctx.JSON(http.StatusOK, api.Reaction{
+					User:     ctx.User.APIFormat(),
```

Comment:
shouldn't this be the reaction.User?

Reply:
Its the same ..

Reply:
but if I use ctx.User I dont have to query the DB

Reply:
Oh yeah sorry